ci/build-subprojects.sh

```diff
@@ -11,7 +11,7 @@ case "$SUBPROJECT" in "rosette")
 
 "core")
 
-    sbt -Dsbt.log.noformat=true clean bnfc:generate coverage test coverageReport
+    sbt -Dsbt.log.noformat=true clean rholang/bnfc:clean rholang/bnfc:generate coverage test coverageReport
```

Comment:
I believe this is unnecessary as there isn't any previously-generated code to clean.  This `SUBPROJECT` is also entirely unrelated to  @jeremybusk's work which is `"rholang_more_tests"` below.

Reply:
As to the bnfc:clean, I can remove it.

As to being unrelated **** sbt and travis. It breaks, and so I'm fixing it.